src/crypto/Crypto.cpp

```diff
@@ -40,7 +40,7 @@ bool Crypto::init()
         return true;
     }
 
-    m_backendVersion = QString::fromLocal8Bit(gcry_check_version(0));
+    m_backendVersion = QString::fromLocal8Bit(gcry_check_version(GCRYPT_VERSION));
```

Comment:
This will cause unintended side effects, please see: https://www.gnupg.org/documentation/manuals/gcrypt/Initializing-the-library.html

If KeePassXC is built on Ubuntu 18.04 and run on 16.04, this check may cause a crash because it will return a nullptr. Technically we should be using a "minimum acceptable version" here.

Reply:
You're right. I'm sorry, I'm changing this one back as well.